aptos-move/framework/aptos-stdlib/sources/cryptography/multi_ed25519.move

```diff
@@ -77,6 +81,33 @@ module aptos_std::multi_ed25519 {
     // Functions
     //
 
+    #[test_only]
+    public fun generate_keys(party_size: u8, threshold: u8): (SecretKey,ValidatedPublicKey) {
```

Comment:
Rename `party_size` to `n`.

Also, can you please flip the order? Typically, we talk about `k` out of `n` multisigs, so this API should first take the threshold `k` and then the number of signers `n`.

Reply:
Also, here, assert if `threshold` <= `n`. Otherwise, abort with an `E_THRESHOLD_TOO_LARGE_OR_ZERO` error code which you can define at the top of the file in the error code section:

```
    /// The threshold must be in the range `[1, n]`, where n is the total number of signers.
    const E_THRESHOLD_TOO_LARGE_OR_ZERO: u64 = 3;
```